spec/facades/quotes_facade_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'rails_helper'
+
+RSpec.describe QuotesFacade do
+  describe '::search(query)' do
+    it 'returns an array of quotes matching a query' do
+      query = 'I am'
+      result = QuotesFacade.search_quotes(query)
+      expect(result.first).to be_a Quote
+      expect(result.first.content).to include(query)
```

Comment:
Nice! I like this assertion.

Reply:
I'm thinking about this assertion more now, and the thing that could get in your way is that the api does a fuzzy search by default. So, the endpoint your hitting could return a quote that had "william" in it because it kindaaaa includes "i am". If that were the case, this assertion would fail.